### DISCUSSION TITLE
new macro recorder app with ctk

### DISCUSSION THREAD

**Discussion by else-wt** [TomSchimansky/CustomTkinter#2624] [Showcase]
Hi there!
I recently created this app using ctk, mainly I would like to receive your opinions about its design, and see if you like the idea. Just that, thanks.
(PS: The READ.ME file is in Spanish, for now you only have to use the page translation tool created by Google, sorry for the inconvenience)
**Link:** https://github.com/else-wt/GSmacro-recorder
![image](https://github.com/user-attachments/assets/37850ecd-c1ce-492c-8d95-23c88a0b72a7)

**Comment by dipeshSam** [comment]
Keep your building ideas up! 🎉
